ui/src/main/javascript/app/core/choosewonder/choosewonder.service.js

```diff
@@ -4,29 +4,29 @@ angular.
     module('sevenWonders.core.choosewonder').
     factory('ChooseWonder', ['$cookies', '$websocket', 'Game', 'Restangular', '$q', '$location',
```

Comment:
You must remove unnecessary dependencies like to be $websocket, $location at the same time the parameters of the function.